### ISSUE TITLE
[Bug] Item Start-slot doesn’t respect inner padding like default- and end-slot

### ISSUE THREAD

**Issue by marvin-jyskebank-dk** [kirbydesign/designsystem#1925]
## Describe the enhancement
The padding top/bot of an item medium is normally 8, unless you put an element in start-slot that is higher then 40px. When you put an element like avatar-medium in the start slot it "eats" the padding

![Skærmbillede 2021-12-10 kl  13 25 59](https://user-images.githubusercontent.com/49033002/145580795-ffb3abf9-87f5-41e6-a834-ca089ed89fca.png)

## Describe the solution you'd like
The start-slot of an item should also respect the items padding (8) 

## Have you considered any alternatives?
Nope

## Are there any additional context?
See screenshot above

Expected behavior should have more padding. see also: 

![Skærmbillede 2021-12-20 kl  08 36 54](https://user-images.githubusercontent.com/49033002/146729743-8c324c2d-1773-4b16-845c-ff964be59e82.png)
<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
This has to do with the fact that we use `--inner-padding-top/bottom` on the item. This means that any padding is only applied to the default slot. To target the whole item, we would have to use `--padding-top/bottom`, but first we need to investigate why this was changed at an earlier point: https://github.com/kirbydesign/designsystem/pull/847/files#diff-ed9293c163f05db6592b5d177a7f9219c243598ebf6ca951c438f59f8a8bf095L12-L13.

**Comment by bommariusser** [comment]
@RasmusKjeldgaard does end slot respect it? not sure it does :(

**Comment by RasmusKjeldgaard** [comment]
The end slot is inside the 'item-inner' element, that gets the padding, so it should actually respect it: 
![image](https://user-images.githubusercontent.com/42470636/151997619-fd062d94-547c-49de-a4c7-c0ccb4182881.png)

**Comment by bommariusser** [comment]
oki doki @RasmusKjeldgaard det kan godt være man så kan lave en snedig workaround på det :D jeg vender tilbage, tak <3

**Comment by RasmusKjeldgaard** [comment]
I think a workaround is to just the avatar some padding on its own in this case, like so:
![image](https://user-images.githubusercontent.com/42470636/152121543-f2226afd-224c-4c83-b9bb-eff028ab0de0.png)

```
<kirby-item>
  <kirby-avatar themeColor="danger" size="md" slot="start" style="padding-top: 8px; padding-bottom: 8px">
    <kirby-icon name="kirby"></kirby-icon>
  </kirby-avatar>
  <kirby-label>
    <h3>Title</h3>
    <p detail>Detail</p>
  </kirby-label>
  <kirby-label slot="end">
    <data>Value</data>
    <data detail>Detail</data>
  </kirby-label>
</kirby-item>
```

**Comment by RasmusKjeldgaard** [comment]
Converting to bug, as this is an issue whenever some start-slot content is larger than content in default/end slot.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
